docs/examples/example.ipynb

```diff
@@ -236,27 +236,237 @@
+    "        arm = cls(config.name)\n",
+    "        return arm\n",
+    "```\n",
+    "\n",
+    "After the component has a defined model and creator function, the component must be registered to the `Registry`. This can be done using [`register_resource_creator()`](https://python.viam.dev/autoapi/viam/resource/registry/index.html#viam.resource.registry.Registry.register_resource_creator) and passing a [`ResourceCreatorRegistration`](https://python.viam.dev/autoapi/viam/resource/registry/index.html#viam.resource.registry.ResourceCreatorRegistration) object with the creator function as a parameter.\n",
+    "\n",
+    "In the same `modular-arm/src` directory, create a new file called `__init__.py`. In it, call `Registry.register_resource_creator()` with the subtype of the component that was subclassed (in this case, `Arm`), the model name, and a `ResourceCreatorRegistration` object with the `new` function defined in the modular resource.\n",
+    "\n",
+    "\n",
+    "```python\n",
+    "# modular-arm/src/__init__.py\n",
+    "from viam.components.arm import Arm\n",
+    "from viam.resource.registry import Registry, ResourceCreatorRegistration\n",
+    "from modular_arm import MyModularArm\n",
+    "\n",
+    "\n",
+    "Registry.register_resource_creator(Arm.SUBTYPE, MyModularArm.MODEL, ResourceCreatorRegistration(MyModularArm.new))\n",
+    "```\n",
+    "\n",
+    "### 3. Create an entry point file\n",
+    "The module now has to be created and started. In the `src` directory, create a new file called `main.py`.\n",
+    "\n",
+    "```python\n",
+    "# modular-arm/src/main.py\n",
+    "import asyncio\n",
+    "\n",
+    "from viam.module.module import Module\n",
+    "from viam.components.arm import Arm\n",
+    "\n",
+    "from .my_modular_arm import MyModularArm \n",
+    "\n",
+    "\n",
+    "async def main():\n",
+    "    \"\"\"This function creates and starts a new module, after adding all desired resources.\n",
+    "    Resources must be pre-registered. For an example, see the `__init__.py` file.\n",
+    "    \"\"\"\n",
+    "\n",
+    "    module = Module.from_args()\n",
+    "    module.add_model_from_registry(Arm.SUBTYPE, MyModularArm.MODEL)\n",
+    "    await module.start()\n",
+    "\n",
+    "\n",
+    "if __name__ == \"__main__\":\n",
+    "    asyncio.run(main())\n",
+    "```\n",
+    "\n",
+    "### 4. Make the module executable\n",
+    "To add the module to your robot, you must provide it as an [executable file](https://docs.viam.com/extend/modular-resources/#make-your-module-executable) that [`viam-server` can access](https://docs.viam.com/extend/modular-resources/#make-sure-viam-server-can-access-your-executable).\n",
+    "\n",
+    "Dependencies for the module (including Viam SDK) have to be installed. In the `modular-arm` directory, create a new file called `requirements.txt` that has all your dependencies. For this example, add `viam-sdk` and `typing_extensions`.\n",
```

Comment:
`typing_extensions` is not required, as it is already a dependency of `viam-sdk`. This should be removed.